extras/check_version.sh

```diff
@@ -25,4 +25,12 @@ if [[ x${PACKAGE_VERSION}x != x${OPENAPI_VERSION}x ]]; then
 	EXITCODE=-1
 fi
 
+# We expect an optional argument containing a version string to be checked against the others
```

Comment:
Is it optional? I feel we should add some piece of documentation at the beginning of this file.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/546/commits/966ac4ebb52dae171ab5ff1cc80ffdf650821554

Reply:
Thanks!